internal/helper/loadbalancermachine.go

```diff
@@ -259,6 +267,51 @@ vrrp_instance ` + VRRPInstanceName + ` {
 	track_process {
```

Comment:
don't you wanted to add advert-int 0.5?

Reply:
Yes it was the plan but this do not work. It must be an INT otherwise it gets rounded 👀